packages/block-library/src/search/block.json

```diff
@@ -34,6 +34,7 @@
 	},
 	"supports": {
 		"align": [ "left", "center", "right" ],
-		"html": false
+		"html": false,
+		"__experimentalBorderRadius": true
```

Comment:
I've noticed the other support flags have been stabilized and no longer have the `__experimental` prefix, so I assume we are happy with the direction and new additions as this can also land as stable cc @jorgefilipecosta @youknowriad @ItsJonQ for thoughts.

Reply:
Also, if we're to expand this with other border properties: does it make sense that the API here allows for grouping things like we do in color?

```
__experimentalBorder: {
    radius: ..., 
    width: ...,
    style: ...
}
```

Reply:
I've updated this to allow for a collection of border related features as per your suggestion. The `__experimental` prefix has been left for the time being.